src/index.js

```diff
@@ -0,0 +1,370 @@
+import React,{Component, useState, useEffect} from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
package imports first, only then internal js files, and only then css in the end